Cython/Debugger/libcython.py

```diff
@@ -117,6 +117,25 @@ def wrapper(self, string, *args, **kwargs):
 # Classes that represent the debug information
 # Don't rename the parameters of these classes, they come directly from the XML
 
+def simple_repr(self, renamed={}, skip=[], state=True):
+    import inspect
+    args = tuple(inspect.signature(self.__init__).parameters)
+    args = tuple(renamed[i] if i in renamed else i for i in args)
+    local = tuple(filter(lambda x: x not in args, vars(self).keys())) \
+        if state else ()
```

Comment:
I'm not sure if we want ordering here in order to get reproducible results?
```suggestion
    local = ()
    if state:
        local = tuple(arg_name for arg_name in vars(self).keys() if arg_name not in args)
```